### DISCUSSION TITLE
截图或者其他应用下载的图片经过传输后日期不正确

### DISCUSSION THREAD

**Discussion by Gausx** [localsend/localsend#441] [L-简体中文]
例如手机A进行截图，此时图片文件日期为5月1日。等到5月3日，通过软件从手机A传输到手机B，此时图片的日期变为了5月3日。而拍照的图片没有影响。通过数据线传输是正常的。

**Comment by FlakyyyCroissant** [comment]
希望可以保留所有传输照片的拍摄日期、地点等信息！而不是更新创建为传输日期，丢失原本照片的信息内容

**Reply by Gausx** [reply]
> 希望可以保留所有传输照片的拍摄日期、地点等信息！而不是更新创建为传输日期，丢失原本照片的信息内容

拍的照片没有事情把  我这边是截图或者保存下来的图片会这样

**Reply by Hitjai** [reply]
拍的照片如果是自動儲存到相簿也會丟失, 手動從紀錄中儲存才不會

**Comment by Mapaler** [comment]
照片的呢拍摄日期是写在EXIF里的，EXIF是图片数据的一部分，所以传输不会丢失。截图没有EXIF，只用文件本身的创建日期或修改日期，这个传输后被改变了自然也就变了。即便是FTP也需要特别勾选保留文件时间线戳才能保留文件时间。

**Comment by yqianning** [comment]
这个问题似乎还没解决，传输录音文件发现数据线传输就不会丢失，修改时间,但是这个就会丢失

**Reply by yqianning** [reply]
> 这个问题似乎还没解决，传输录音文件发现数据线传输就不会丢失，修改时间,但是这个就会丢失

经过测试，手机文件直接点分享，分享到localsend在传输就会丢失。从这个软件本身选择文件传输就不会丢失
